### PR TITLE
Feature/due date

### PR DESCRIPTION
**Pull request by cjrpostma**
# Description :: User Story

- Add DueDate component and subscribe it to data from Playlist.
- Test correct rendering.

## Type of change

- [x] New Feature
- [ ] Bug Fix
- [ ] Refactor
- [ ] Breaking Change

## Test Results

- Pass

### CODE REVIEW

File: src/components/DatePickerInput/DatePickerInput.js [hunk 1]
```diff
@@ -14,7 +14,6 @@ const DatePickerInput = ({ playlistDate, setPlaylistDate }) => {
       format="MM/DD/YYYY"
       margin="normal"
       id="date-picker-inline"
-      inputValue={playlistDate}
```
Comment (ryanbahan) [inline src/components/DatePickerInput/DatePickerInput.js:17]: Why did this get deleted?

Comment (ryanbahan) [reply src/components/DatePickerInput/DatePickerInput.js:17]: We need an inputValue and value for the datepicker, as far as I'm aware

File: src/components/DueDate/DueDate.test.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import { ThemeProvider } from 'styled-components';
+import { MuiPickersUtilsProvider } from '@material-ui/pickers';
+import MomentUtils from '@date-io/moment';
+import theme from '../../styles/theme';
+import UserProvider from '../../contexts/userContext';
+import DueDate from './DueDate';
```
Comment (ryanbahan) [inline src/components/DueDate/DueDate.test.js:9]: nice job adding some tests

File: src/components/DatePickerInput/DatePickerInput.js (hunk 1 above)
Comment (cjrpostma) [reply src/components/DatePickerInput/DatePickerInput.js:17]: After testing with and without this prop, it worked either way. Happy to add back if I'm missing something. The initial value and input both feed from the value prop.

Comment (ryanbahan) [reply src/components/DatePickerInput/DatePickerInput.js:17]: Let me pull this down and double check

Comment (ryanbahan) [reply src/components/DatePickerInput/DatePickerInput.js:17]: Looks like you're right. Not sure what was making that behavior different initially. Nice job!

**Pull request merged**